application/dash_application/views/single_lane_ts.py

```diff
@@ -524,6 +527,15 @@ def update_pressed(click,
 
         (jira_href, jira_style) = sidebar_utils.jira_display_button(runs, title)
 
+        (miso_request, miso_button_style) = util.build_miso_info(df, title, 
+            [{
+                'title': 'Passed Filter Reads (*10^6)',
```

Comment:
Should this be `Total Clusters`?

Reply:
Looks like part of SLTS got overlooked when making this switch. I'll fix it in another PR then rebase this and fix